arrow-cast/src/cast.rs

```diff
@@ -279,6 +283,9 @@ pub fn can_cast_types(from_type: &DataType, to_type: &DataType) -> bool {
 ///   in integer casts return null
 /// * Numeric to boolean: 0 returns `false`, any other value returns `true`
 /// * List to List: the underlying data type is cast
+/// * List to FixedSizeList: the underlying data type is cast. If safe is true and a list element
+/// has the wrong length it will be replaced with NULL, otherwise returing an error
+/// it will be replaced with null if safe is true, otherwise an error will be returned
```

Comment:
Is this what you meant?
```suggestion
/// it will be replaced with null if safe is true, otherwise an error will be returned.
```

Reply:
Something like that, been a long week :sweat_smile: